### PR TITLE
Draft Feed Page

### PR DESCRIPTION
**Pull request by Archna-1**
One card with dummy information. Can add more cards later with info from database.

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -14,9 +15,9 @@ app.use(bodyParser.json());
 mongoose.connect(process.env.MONGO_URL, { useNewUrlParser: true, useUnifiedTopology: true });
 var db = mongoose.connection;
 
-const Profile = require("./models/profile.js");
-const Likes = require("./models/Likes.js");
-const User = require("./models/User.js");
+const Profilerouter = require("./routes/profiles.js");
```
Comment (alicesf2) [inline backend/index.js:18]: Now that it's working, i'm gonna set up a separate routes.js file to clean this up a bit :)

File: backend/routes/profiles.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+const router = express.Router();
+
+const { Profile } = require('../models');
+
+router.get("/", async (req, res) => {
+    const response = await Profile.find({});
+    res.json(response);
+  });
+
+  module.exports = router;
```
Comment (alicesf2) [inline backend/routes/profiles.js]: nitpick: remove the indent for this line, also i can show u guys some auto-formatting tools that will make everything formatted when u save the file

File: frontend/src/pages/FeedPage.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+import '../styles/FeedPage.css';
+
+import { Card, Image, Button, Icon, Divider, FeedExtra } from 'semantic-ui-react'
+import { getProfiles } from "../utils/api";
+
+function FeedPage() {
+
+  const [ profiles, setProfile] = useState([]);
+  useEffect(() => {
+    const demoCall = async () => {
```
Comment (alicesf2) [inline frontend/src/pages/FeedPage.js]: Maybe rename demoCall to something more specific

File: frontend/src/pages/FeedPage.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      const profiles = await getProfiles();
+      console.log(profiles);
```
Comment (alicesf2) [inline frontend/src/pages/FeedPage.js]: Make sure to remove console.logs when ur done testing

File: frontend/src/utils/api.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+export async function getProfiles() {
```
Comment (alicesf2) [inline frontend/src/utils/api.js:1]: Remove console.logs from this function as well

File: frontend/src/pages/FeedPage.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+      setProfile(profiles);
+    };
+    demoCall();
+  }, []);
+
+  return (
+    <div class="feedpage">
+      {profiles.map(value =>
```
Comment (alicesf2) [inline frontend/src/pages/FeedPage.js:20]: yess this is lit :))

File: frontend/src/pages/FeedPage.js [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+        <Card.Group itemsPerRow={1} style ={{display: 'flex', marginLeft: 'auto', marginRight: 'auto'}}>
```
Comment (alicesf2) [inline frontend/src/pages/FeedPage.js:21]: usually classes + css files are preferred over inline styling, but this is fine for product showcase

**Review by alicesf2 — CHANGES_REQUESTED** [review]
Looks awesome, good work! Once u address the comments I've made, repush ur code and I'll approve 👍

File: frontend/src/pages/FeedPage.js (hunk 7 above)
Comment (coconut750750) [reply frontend/src/pages/FeedPage.js:21]: "fine for product showcase" lol

**Comment by coconut750750** [comment]
Good work Archna!

**Review by coconut750750 — APPROVED** [review]

File: frontend/src/pages/FeedPage.js (hunk 3 above)
Comment (Archna-1) [reply frontend/src/pages/FeedPage.js:11]: Would loadProfiles be a better name?

Comment (Archna-1) [reply frontend/src/pages/FeedPage.js]: > Maybe rename demoCall to something more specific

Would loadProfiles be a better name?

Comment (alicesf2) [reply frontend/src/pages/FeedPage.js]: yea that would be good!

**Review by alicesf2 — APPROVED** [review]
Awesome job :starchna:

**Pull request merged**